src/core/api/option_utils.ts

```diff
@@ -667,7 +665,6 @@ function parseLoadVideoOptions(
 
   // remove already parsed data to simplify the `transportOptions` object
   delete transportOptions.initialManifest;
-  delete transportOptions.manifestUpdateUrl;
```

Comment:
I was wondering why this commit would work haha.

That's one of the magic lines.